networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,83 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-4"]
+== Network Observability Operator 1.4.0
+The following advisory is available for the Network Observability Operator 1.4.0:
+
+* https://errata.devel.redhat.com/advisory/116729
+ https://access.redhat.com/errata/RHSA-2023:1817[RHSA-2023:1817 Network Observability Operator 1.4.0]
+
+[id="network-observability-channel-deprecation-1.4"]
+=== Channel Deprecation
```

Comment:
Do you make a difference in doc between "deprecation" and "removal"? I'm used to seeing "deprecation" as something like in a grace period before removal; which, if I'm seeing it correctly, wouldn't be the case now, it's really removed.

Reply:
Ah yeah this is corect. I see have guidance about the stages: https://redhat-documentation.github.io/supplementary-style-guide/#_deprecated_functionality

This is my first time actually going through a removal of any kind of feature, over time, so this is new to me.